<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);
border-radius: 10px ; 
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>                    Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px ;
background-color: rgb(255, 255, 255) ;</string>
     </property>
     <property name="text">
      <string>User-Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px ;
background-color: rgb(255, 255, 255) ;</string>
     </property>
     <property name="text">
      <string>PassWord:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px ;
background-color: rgb(255, 255, 255) ;</string>
     </property>
     <property name="text">
      <string>Confirm PassWord:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSignUp">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>390</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 54, 90);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>450</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(35, 186, 210);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>I already have account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../Pictures/Roblox/logo-removebg-preview.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
